noxfile.py

```diff
@@ -110,11 +111,25 @@ def lint_setup_py(session):
 def compliance_test(session):
     """Run SQLAlchemy dialect compliance test suite."""
     session.install("pytest")
+    session.install("mock")
     session.install("-e", ".")
     session.run("python", "create_test_database.py")
     session.run("pytest", "-v")
 
 
+@nox.session(python=DEFAULT_PYTHON_VERSION)
+def unit(session):
+    """Run unit tests."""
+    # Run SQLAlchemy dialect compliance test suite with OpenTelemetry.
+    session.install("pytest")
+    session.install("mock")
+    session.install("-e", ".")
+    session.install("opentelemetry-api==1.1.0")
+    session.install("opentelemetry-sdk==1.1.0")
+    session.install("opentelemetry-instrumentation==0.20b0")
+    session.run("py.test", "--quiet", os.path.join("test"), *session.posargs)
```

Comment:
Strangely, this seemed to run the compliance suite tests as well 😕 Perhaps create a `test/unit` folder, move the OT tests, and update the path. That way if we add more unit test files in future it's very clear where to put them and they'll run correctly until this session 👍

Reply:
Moved it to `test/unit/` Also had to change to `fixtures.TestBase` as base class for `OpenTelemetryBase` instead of `unittest.Testcase` as the later caused the tests to be skipped.